spec/factories/users.rb

```diff
@@ -8,5 +8,6 @@
     email { Faker::Internet.safe_email }
     password { Faker::Internet.password(min_length = 10, max_length = 20, mix_case = true, special_chars = true) }
     password_confirmation { password }
+    association :user_profile, factory: :user_profile
   end
 end
```

Comment:
why does this factory looks so different compared to user_profiles?

Reply:
Can you elaborate?

Reply:
Nevermind github show me just lines 8-12 and i did not understand how it worked without the missing stuff^^